tests/foreman/cli/test_activationkey.py

```diff
@@ -1368,4 +1361,28 @@ def test_positive_content_override(self):
                     u'organization-id': self.pub_key['org_id'],
                     u'value': override_value,
                 })
-                self.assertIn(result[0], 'Updated content override')
+                # Retrieve the product content enabled flag
+                result = ActivationKey.product_content({
```

Comment:
product_content don't return a CSV output? I think we should make it return a proper output and let the already in place parser take care of it.